devel/libprotozero/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                    1.0
+PortGroup           cmake                     1.1
+PortGroup           github                    1.0
+
+name                libprotozero
+
+github.setup        mapbox protozero 1.7.0 v
+
+categories          devel
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+license             BSD Boost-1 Apache-2
+
+description         A minimalistic protocol buffer decoder and encoder in C++
+
+long_description    Low-level: this is designed to be a building block \
+                    for writing a very customized decoder for a stable \
+                    protobuf schema. If your protobuf schema is \
+                    changing frequently or lazy decoding is not \
+                    critical for your application then this approach \
+                    offers no value: just use the C++ API that can be \
+                    generated with the Google Protobufs protoc \
+                    program.
+
+homepage            https://github.com/mapbox/protozero
```

Comment:
this is the default that is set by the github PG based on the information you provided in the `github.setup` line. Therefore, it is redundant and you don't need to manually set it (again).

Reply:
Thank you.  I'll remove.